PixelToolkit.py

```diff
@@ -11,13 +11,29 @@
 
     pass_gen = subparsers.add_parser("pass-gen", help="Password Generator")
     pass_gen.add_argument("--length", "-l", type=int, help="Length of the password", default=10)
+    
+    port_scan = subparsers.add_parser("port-scan", help="Port Scanner")
+    port_scan.add_argument("--host", type=str, help="Host to scan", default="127.0.0.1")
```

Comment:
Short `-h` could also be included

Reply:
there's an issue with this. So I need to set add_help in add_parser to false, because every other way will result in error with '-h' included.
raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument --host/-h: conflicting option string: -h

Reply:
ohh right. we need add_help so don't do that. I guess we could use something like a `free floating argument` as a alternative. I may look into doing that later though. That's fine for now